src/test/regress/expected/rollback_to_savepoint.out

```diff
@@ -0,0 +1,27 @@
+-- Regression test for this issue:
+-- https://github.com/citusdata/citus/issues/3622
+SET citus.shard_count = 4;
+SET citus.next_shard_id TO 1954000;
+CREATE SCHEMA rollback_to_savepoint;
+SET search_path TO rollback_to_savepoint;
+CREATE TABLE t(a int);
+SELECT create_distributed_table('t', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- This timeout is chosen such that the INSERT with
+-- generate_series(1, 100000000) is cancelled at the right time to trigger the
+-- bug
+SET statement_timeout = '2s';
```

Comment:
Hmm, what if in 2 seconds Citus cannot start COPY command yet (e.g., still collecting the results of SELECT)? Would we still trigger the bug?

Can we maybe convert this to a failure tests where we fail once COPY starts?

Reply:
I wasn't able to reproduce the original issue with our failure testing suite. So I'm keeping it as is.